src/CommandLineArgumentsParser/Messages.cs

```diff
@@ -20,6 +20,7 @@ internal static class Messages
     public static string EXC_ARG_NOT_ONE_WORD = "LongName of an argument must be one word.";
 
     public static string EXC_ARG_SWITCH_PRINT = "Argument: {0} value: {1}";
+    public static string IL_EXC_ARG_SWITCH_PRINT = "Argument: {@Argument} value: {@Value}";
```

Comment:
I think you used an older branch, because some of these defines were deleted.

So please make sure you branch from the master.

Reply:
I used 3.0.23 as baseline for making the changes.